spec/controllers/support_ticket/academy_details_controller_spec.rb

```diff
@@ -3,6 +3,7 @@
 RSpec.describe SupportTicket::AcademyDetailsController, type: :controller do
```

Comment:
Please could you add a test case for this change? Perhaps just testing the method in `BaseController` directly would be easier.